FirebaseAuthUI/FIRAuthUIBaseViewController.h

```diff
@@ -104,6 +104,12 @@ typedef void (^FIRAuthUIAlertActionHandler)(void);
  */
 - (void)pushViewController:(UIViewController *)viewController;
 
+/** @fn onBackButtonPressed:
+    @brief Pops the view controller from navigation stack. If current controller is root 
+    works as @c cancelAuthorization
+ */
+- (void)onBackButtonPressed;
```

Comment:
I think developers may be hesitating what is class type of Button (UIButton/UINavItem or they even can use voice command) and if it's eligible to pass it.
We are not using this value, so I think it's better just to rename to something like this:
```
- (void)onBack
- (void)navigateBack
- (void)goBack
```

What do you think?